src/renderer/components/RenderMedia.js

```diff
@@ -0,0 +1,29 @@
+const React = require('react')
+
+class RenderMedia extends React.Component {
+  render () {
+    var { className, url, contentType } = this.props
```

Comment:
Maybe `const { className, url, contentType = 'image/jpg' } = this.props` and remove if-statement below?

Reply:
tried something different instead